cg/store/models.py

```diff
@@ -797,6 +798,8 @@ class Sample(Base, PriorityMixin):
         back_populates="sample", cascade="all, delete"
     )
 
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
This seems off to me. This is not a BaseModel right?

Reply:
Exactly, this `Base` is from SQLAlchemy `DeclarativeBase` not from Pydantic `BaseModel`. Not sure what is the purpose here @beatrizsavinhas

Reply:
Unfortunately forgotten line of code from when I was trying to understand the validation errors I was getting. Removed now!